docs/make_docs.sh

```diff
@@ -3,17 +3,3 @@
 rm -f 4-Commands/*.md
 
 go run docs.go
```

Comment:
Now that this file became only two commands, I guess would be good to change remove this file and leave them only on kool.yml?

```yml
  make-docs:
    - rm -f docs/4-Commands/*.md
    - kool run go run docs/docs.go
```

PS: we might even move `docs.go` to the root folder, and then add a `+build ignore` annotation to it.